nextflow_schema.json

```diff
@@ -52,20 +54,13 @@
                 "fasta": {
```

Comment:
I'm confused. Do you need `fasta` or not? If not, remove here as well

Reply:
yes we need the `fasta`